aqua-examples/ts-oracle/Readme.md

```diff
@@ -6,98 +6,153 @@ Getting accurate timestamps can be problematic in various contexts including blo
+result: Object({"err_str": String(""), "freq": Number(2), "mode": Number(1661345701640), "n": Number(11)})
+ elapsed time: 8.608047ms
+
+3> q
 ```
 
-test the code with
+On the 1st step, we explored our module interface and data using the `i` command, on the 2nd we made a call of the `point_estimate` function from the `ts_oracle` interface.
+
+If you navigate to our module directory `ts-oracle/modules/ts-oracle`, you can also unit test the code with
 
 ```bash
 cargo +nightly test --release
 ```
 
-and deploy the service to a peer of your choice with the `aqua` cli tool:
+It generates the output similar to:
+
+```
+....
+running 6 tests
+test tests::test_mean_good ... ok
+test tests::test_mean_bad ... ok
+test tests::test_mode ... ok
+test tests::test_point_estimate_bad_2 ... ok
+test tests::test_point_estimate_bad ... ok
+test tests::test_point_estimate_good ... ok
+
+test result: ok. 6 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.35s
+```
+
+For deploying the service to a peer with the `fluence` CLI tool, you need to run from the project root:
 
 ```bash
-aqua remote deploy_service \
-     --addr krasnodar-06 \
-     --config-path configs/ts_oracle_deploy_cfg.json \
-     --service ts-oracle \
-     --sk <your secret key>
+fluence deploy
 ```
 
 which results in:
 
 ```bash
-Your peerId: 12D3KooWHS2Eys5GmmdBGZHwUGRM4TRvcaW4m7dgGAShxbMEcKms
-"Going to upload a module..."
-2022.02.08 14:30:48 [INFO] created ipfs client to /ip4/161.35.212.85/tcp/5001
-2022.02.08 14:30:48 [INFO] connected to ipfs
-2022.02.08 14:30:50 [INFO] file uploaded
-"Now time to make a blueprint..."
-"Blueprint id:"
-"5617e051fbd7f8e18dfc16cace62f71f0ff090d576ee09e3520c196ef44f6a79"
-"And your service id is:"
-"388d8717-b547-43c3-af05-b4713893d453"
+Making sure all services are downloaded... done
+Making sure all modules are downloaded and built... done
+
+Going to deploy services described in <path-to-examples>/aqua-examples/ts-oracle/fluence.yaml:
+
+tsOracle:
+  get: ./ts-oracle
+  deploy:
+    - deployId: default
+
+
+? Do you want to deploy all of these services? Yes
+Deploying:
+  service: tsOracle
+  deployId: default
+  on: 12D3KooWCMr9mU894i8JXAFqpgoFtx6qnV1LFPSfVc3Y34N4h4LS
+... done
+Compiling <path-to-examples>/aqua-examples/ts-oracle/.fluence/aqua/deployed.app.aqua... done
+
+Currently deployed services listed in <path-to-examples>/aqua-examples/ts-oracle/.fluence/app.yaml:
+
+tsOracle:
+  default:
+    - blueprintId: b2325812dc59af855ac7d1ea45ce12fb033822dc88c8a26425007bfd4eafae1c
+      serviceId: 605b227c-e67a-4888-a65a-60df6a1fb862
+      peerId: 12D3KooWCMr9mU894i8JXAFqpgoFtx6qnV1LFPSfVc3Y34N4h4LS
+
 ```
 
-As always, take note of the service id!
+The tool asked us if we want to deploy the service, and after confirming it, we have our service successfully deployed. Please also note in the output that we got the wrapper for our service environment generated in the `./.fluence/aqua/deployed.app.aqua` file.
```

Comment:
if you specify `fluence deploy` why do you need to confirm the option?

Reply:
I don't know, seems like a redundant. I believe it was a recent change after introduction deployment strategies.